_posts/en/newsletters/2019-02-05-newsletter.md

```diff
@@ -0,0 +1,283 @@
+---
+title: 'Bitcoin Optech Newsletter #32'
+permalink: /en/newsletters/2019/02/05/
+name: 2019-02-05-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter includes an announcement of the 2019 Chaincode
+Residency program, summarizes a few talks from the Stanford Blockchain
+Conference, and provides the usual list of notable code changes in
+popular Bitcoin infrastructure projects.
+
+## Action items
+
+- **Apply to the Chaincode Residency:** - Bitcoin Optech encourages any
+  engineer who is interested in spending summer contributing to open source
+  Bitcoin and Lightning projects to apply to the Chaincode Residency. Full
+  details of the residency are in the *News* section below.
+
+## News
+
+- **2019 Chaincode Residency:** Chaincode Labs opened applications for its
+  [fourth residency program][residency] to be held in New York over
+  summer 2019. The program combines a 3 week seminar and discussion series
+  covering Bitcoin and Lightning protocol development with a two month period for
+  working on an open source Bitcoin or Lightning project under the guidance of an
+  established protocol developer. The list of [confirmed speakers and
+  mentors][residency speakers] includes some of the most prolific contributors to
+  Bitcoin and Lightning. The previous Chaincode residency (focused on Lightning
+  Network applications) was covered in [Newsletter #21][].
+
+    Chaincode is inviting developers who want to contribute to open source
+    Bitcoin and Lightning protocol projects to [apply to the
+    residency][residency apply]. Applicants from all backgrounds are welcomed,
+    and Chaincode will cover travel and accommodation costs and provide a stipend
+    to support living expenses.
+
+[residency]: https://residency.chaincode.com
+[residency speakers]: https://residency.chaincode.com/#mentors
+[residency apply]: https://residency.chaincode.com/#apply
+
+## Notable talks from the Stanford Blockchain Conference
+
+The [third edition][sbc] of this annual conference (formerly named BPASE)
+included more than two dozen presentations across three days.  We found
+the following topics especially interesting from a Bitcoin perspective,
+although we encourage anyone who wants to learn more to look at the
+[transcripts][] provided by Bryan Bishop or the videos provided by
+the organizers ([day 1][], [day 2][], [day 3][]).
+
+- **Accumulators for blockchains** presented by Benedickt Bünz
+  ([transcript][accumulators txt], [video][accumulators vid]).  Bitcoin
+  full nodes maintain a ledger (called the UTXO set) that stores the
+  ownership information for every currently-spendable grouping of
+  bitcoins.  Currently, that ledger contains over 50 million entries
+  and uses about 3.8 gigabytes of disk space.  This allows nodes who
```

Comment:
https://statoshi.info/dashboard/db/unspent-transaction-output-set says 2.76GiB. Any idea where the discrepancy comes from?

Reply:
At the current height, `gettxoutsetinfo` on my node returns the following two fields:

```json
  "bogosize": 3814123285,
  "disk_size": 2971132652
```

I previously went with bogosize, which is a deterministic size that all nodes should report the same when on the same chaintip.  Looks like Lopp is using disk_size, which can vary between different nodes.  I think there's an argument to be made for bogosize being the true size of the data, but given that the disk size is what people really care about (and what affects performance the most), I pushed a commit updating this to "about three gigabytes".